docs.phtml

```diff
@@ -31,25 +31,25 @@
     <li><a href="<?=$top_dir;?>/viewdoc.phtml?rev=<?=$major_stable;?>&amp;lang=C&amp;doc=guide"><b><?php echo T_("The Tutorial and Concepts Guide");?></b></a></li>
   </ul>
 
-  <p class="justify"><?php echo T_("The <b>Help Manual</b> is designed to be a quick reference of how to accomplish specific tasks and how to use the features in <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is designed to be an in depth guide to the concepts behind using <span class=\"gnucash\">GnuCash</span> with a tutorial to show how to put those concepts into practice.");?></p>
+  <p class="justify"><?php echo T_("The <b>Help Manual</b> details specific tasks and features of <span class=\"gnucash\">GnuCash</span>, while the <b>Concepts Guide</b> is an in-depth guide.");?></p>
```

Comment:
It is actually the [GnuCash Tutorial and Concepts Guide](https://gnucash.org/viewdoc.phtml?rev=4&lang=C&doc=guide)
Could it be renamed just "Concepts"?

Reply:
If you're following official descriptive cataloging rules, then the Help Manual is "actually" the "GnuCash Help Manual" and go ahead changing that throughout the realm while you're at it.

The fact is, the document in question had been referred to as the Tutorial & Concept Guide for as long as I've known about it.

Reply:
There is no ampersand, and it is "concepts". It is referred to as  both "The Concepts Guide" and "The Tutorial and Concepts Guide" in https://www.gnucash.org/docs.phtml Those two variants stand out in bold, and they are not links, but instead the user is meant to know both of those refer to what was earlier described as how many parts it is divided into.

In the interest of the "we" in

> We want feedback from you, it is also through your comments that we know how to improve the documentation

> The Concepts Guide is designed to be an in depth guide to the concepts behind using GnuCash with a tutorial to show how to put those concepts into practice.

It says "to the concepts".
That doesn't bring anything beyond the title, which is already "Concepts", other than concepts having the quality of being "behind using GnuCash".
The concepts (double reference too), can be put into practice by way of "Concepts" the document being a guide.
I don't see a lot of helpful information beyond that.
There is a document called concepts, it is a guide, the guide is an in-depth one.

Why it being designed that way is important alludes me.
An in-depth guide amounting to a tutorial is a given, and maybe that is useful info somehow. If you don't know what a guide is maybe? I haven't read it, so I really don't know.
With that in mind, it is hard for me to get what is being communicated here.

It seems in a roundabout way an in-depth guide called Concepts is being presented as such, with a lot of non-supporting language to muddle things up.

From my cursory understanding of English and the GnuCash Concepts and Tutorial Guide, it isn't actually a guide(?)

[A tutorial](https://en.wikipedia.org/wiki/Tutorial), in education, is a method of transferring knowledge and may be used as a part of a learning process. More interactive and specific than a book or a lecture, a tutorial seeks to teach by example and supply the information to complete a certain task.

Edit: The help seems to be the same type of guide, covering some of the same topics, but with more info on specifics.
"Reports and Charts" says 

> This section is a “work in process”. Some of the material has not been reviewed for V2.0. While it may not be strictly accurate, it is at least a "guide".

That is at-least 10 years out of date (2.0), but maybe more like 15 (1.9.6).

Reply:
Wow. Talk about picky! I'm done with you. 

@fellen: this is why I dropped out of this community. I'll once again depart this loonie bin.

Reply:
> Wow. Talk about picky! I'm done with you.
> 
> @fellen: this is why I dropped out of this community. I'll once again depart this loonie bin.

And once again I'm sad to see this happen...